clang/docs/LanguageExtensions.rst

```diff
@@ -1715,11 +1715,41 @@ Variadic Friends                              __cpp_variadic_friend            C
 Trivial Relocatability                        __cpp_trivial_relocatability     C++26         C++03
 --------------------------------------------- -------------------------------- ------------- -------------
 Designated initializers (N494)                                                 C99           C89
+``_Complex`` (N693)                                                            C99           C89, C++
+``_Bool`` (N815)                                                               C99           C89
+Variable-length arrays (N683)                                                  C99           C89, C++
+Flexible array members                                                         C99           C89, C++
+static and type quals in arrays                                                C99           C89
+``long long`` (N601)                                                           C99           C89
+Universal character names                                                      C99           C89
```

Comment:
Is this actually the case? We error on e.g. `int e\u00fe = 4;` in C89 mode (https://godbolt.org/z/GKqx78xzM):
```console
<source>:3:10: warning: universal character names are only valid in C99 or C++; treating as '\' followed by identifier [-Wunicode]
    3 |     int e\u00fe = 4;
      |          ^
<source>:3:10: error: expected ';' at end of declaration
    3 |     int e\u00fe = 4;
      |          ^
      |          ;
```

Reply:
I tried each of these as well and I thought that worked, but I think I got confused because we emit the pedantic diagnostic and then fail to lex properly anyway. I'll remove this one from the list, good catch!